guides/keras_cv/simsiam-with-kerascv.py

```diff
@@ -0,0 +1,636 @@
+"""
+Title: SimSiam Training with TensorFlow Similarity and KerasCV
+Author: [lukewood](https://lukewood.xyz), Ian Stenbit, Owen Vallis
+Date created: 2022/09/21
+Last modified: 2022/09/21
+Description: Train a KerasCV model using unlabelled data with SimSiam.
+"""
+
+"""
+## Overview
+
+[TensorFlow similarity](https://github.com/tensorflow/similarity) makes it easy to train
+KerasCV models on unlabelled corpuses of data using contrastive learning algorithms such
+as SimCLR, SimSiam, and Barlow Twins.  In this guide, we will train a KerasCV model
+using the SimSiam implementation from TensorFlow Similarity.
+
+## Background
+
+Self-supervised learning is an approach to pre-training models using unlabeled data. T
+his approach drastically increases accuracy when you have very few labeled examples but
```

Comment:
I guess this probably renders fine but there is a linebreak in the middle of a word here

Reply:
fixed